apps/predictions/lib/prediction.ex

```diff
@@ -25,6 +27,9 @@ defmodule Predictions.Prediction do
           stop: Stops.Stop.t() | nil,
           route: Routes.Route.t(),
           direction_id: 0 | 1,
+          arrival_time: DateTime.t() | nil,
+          departure_time: DateTime.t() | nil,
+          # TODO: Deprecated, should be removed in favor of arrival_time and departure_time -- MSS 2022-09-22
```

Comment:
**suggestion:** I'm not sure how your team tracks this sort of cleanup work, but if this field should be removed eventually is there a ticket to represent that?

Reply:
There is now! :D